Makefile

```diff
@@ -48,38 +56,41 @@ clean-dockerdemo:
 ssh:
 	@vagrant ssh netplugin-node1 || echo 'Please run "make demo"'
 
-unit-test: build
-	CONTIV_HOST_GOPATH=$(GOPATH) CONTIV_HOST_GOBIN=$(HOST_GOBIN) \
-					   CONTIV_HOST_GOROOT=$(HOST_GOROOT) ./scripts/unittests -vagrant
+unit-test: stop clean build
```

Comment:
In short, this is quite a bit slower but I think it's worth the trouble. Without it, the build will break between multi-os tests.

If you build on centos, you're going to get a centos-friendly binary in `bin`. Ubuntu will differ. The `clean` ensures the build does not break or produce false-positives because of an object mismatch.

I don't know what the HOST flags are solving here. The goal is to make our CI line up with our real-world environments. Host flags like this change the build toolchain, which may not be what's in the VM.